src/game/stage/camera.c

```diff
@@ -0,0 +1,645 @@
+    [LEVEL_INDEX(ZONE_3, ACT_BOSS)] = StageBgUpdateZone3ActBoss,
+    [LEVEL_INDEX(ZONE_3, ACT_UNUSED)] = stageBgUpdateDummy,
+
+    // Zone 4
+    [LEVEL_INDEX(ZONE_4, ACT_1)] = StageBgUpdateZone4Acts12,
+    [LEVEL_INDEX(ZONE_4, ACT_2)] = StageBgUpdateZone4Acts12,
+    [LEVEL_INDEX(ZONE_4, ACT_BOSS)] = stageBgUpdateDummy,
+    [LEVEL_INDEX(ZONE_4, ACT_UNUSED)] = stageBgUpdateDummy,
+
+    // Zone 5
+    [LEVEL_INDEX(ZONE_5, ACT_1)] = StageBgUpdateZone5Acts12,
+    [LEVEL_INDEX(ZONE_5, ACT_2)] = StageBgUpdateZone5Acts12,
+    [LEVEL_INDEX(ZONE_5, ACT_BOSS)] = StageBgUpdateZone5ActBoss,
+    [LEVEL_INDEX(ZONE_5, ACT_UNUSED)] = stageBgUpdateDummy,
+
+    // Zone 6
+    [LEVEL_INDEX(ZONE_6, ACT_1)] = StageBgUpdateZone6Acts12,
+    [LEVEL_INDEX(ZONE_6, ACT_2)] = StageBgUpdateZone6Acts12,
+    [LEVEL_INDEX(ZONE_6, ACT_BOSS)] = StageBgUpdateZone6ActBoss,
+    [LEVEL_INDEX(ZONE_6, ACT_UNUSED)] = stageBgUpdateDummy,
+
+    // Zone 7
+    [LEVEL_INDEX(ZONE_7, ACT_1)] = StageBgUpdateZone7Acts12,
+    [LEVEL_INDEX(ZONE_7, ACT_2)] = StageBgUpdateZone7Acts12,
+    [LEVEL_INDEX(ZONE_7, ACT_BOSS)] = StageBgUpdateZone7ActBoss,
+    [LEVEL_INDEX(ZONE_7, ACT_UNUSED)] = stageBgUpdateDummy,
+
+    // Zone Final
+    [LEVEL_INDEX(ZONE_FINAL, ACT_XX_FINAL_ZONE)] = StageBgUpdateZoneFinalActXX,
+    [LEVEL_INDEX(ZONE_FINAL, ACT_TRUE_AREA_53)] = StageBgUpdateZoneFinalActTA53,
+    [LEVEL_INDEX(ZONE_FINAL, ACT_BOSS)] = StageBgUpdateZone1Acts12,
+    [LEVEL_INDEX(ZONE_FINAL, ACT_UNUSED)] = StageBgUpdateZone2Acts12,
+
+    // Zone Unused
+    [LEVEL_INDEX(ZONE_UNUSED, ACT_1)] = StageBgUpdateZone1Acts12,
+    [LEVEL_INDEX(ZONE_UNUSED, ACT_2)] = StageBgUpdateZone6Acts12,
+};
+
+static const s8 gUnknown_080D5A98[NUM_LEVEL_IDS][4] = {
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x40, 0x02, 0x1C }, //
+    { 0x20, 0x40, 0x02, 0x1C }, //
+    { 0x20, 0x40, 0x02, 0x1C }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x40, 0x02, 0x1C }, //
+    { 0x20, 0x40, 0x02, 0x1C }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x40, 0x02, 0x1C }, //
+    { 0x40, 0x20, 0x02, 0x1C }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1C }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1C }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x20, 0x40, 0x02, 0x1C }, //
+    { 0x20, 0x20, 0x02, 0x1D }, //
+    { 0x40, 0x20, 0x02, 0x1C }, //
+};
+
+void sub_801C068(u32 level)
```

Comment:
CreateCamera?

Reply:
Or maybe `InitCamera`?

Reply:
Yeah, I thought that could make sense as well but didn't just wanna do it without being 100% sure...